gradient_utils/metrics.py

```diff
@@ -21,8 +21,20 @@ def _get_env_var_or_raise(*env_vars):
     return rv
 
 
+def _get_experiment_id():
+    if os.getenv('PAPERSPACE_EXPERIMENT_ID'):
+        return os.getenv('PAPERSPACE_EXPERIMENT_ID')
+    hostname = os.getenv('HOSTNAME')
+    try:
+        experiment_id = hostname.split('-')[1]
+        return experiment_id
+    except IndexError:
+        msg = "{} experiment ID not found"
```

Comment:
What's with the {}?

Reply:
Also: if there is no HOSTNAME env var, os.getenv will return None and it will crash on None.split()

Reply:
Thank you, make sense. Fixing